### ISSUE TITLE
[Spotter] Search input does not have a context menu

### ISSUE THREAD

**Issue by pavel-krivanek** [pharo-spec/NewTools#198]
The Search input in Spotter does not react on RMB to open context menu with clipboard operations etc.

**Comment by estebanlm** [comment]
Yeah, that's because `SptextinputPresenter` does not implements it by default. I will need to think how to add it.
